packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1167,9 +1181,18 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             disableIsolatedChannels: metadata?.disableIsolatedChannels,
             gcVersion: metadata?.gcFeature,
         });
+
         ReportOpPerfTelemetry(this.context.clientId, this.deltaManager, this.logger);
     }
 
+    public checkNavigator(): boolean {
```

Comment:
public functions can't be optimized by bundler (needs to keep same long name). I'm not sure why it has to be public.
I'd at least make it private, better - inline into the only call site where it's used (code would be smaller).